### PR TITLE
use refactored find a pat script to pass token to slash command dispatch action via env

### PR DESCRIPTION
**Pull request by zzztimbo**
## Summary 
This reinstates the work of @tuliren's [PR](https://github.com/airbytehq/airbyte/pull/16144) by using a refactored `find_non_rate_limited_PAT` that writes the token to env (not base64 encoded). 

## Tests 
- https://github.com/airbytehq/github-workflow-test-repo-base/runs/8173224835?check_suite_focus=true
- https://github.com/airbytehq/github-workflow-test-repo-base/runs/8173220486?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/slash-commands.yml [hunk 1]
```diff
@@ -8,17 +8,29 @@ jobs:
...
           pr_info="$(curl ${{ github.event.issue.pull_request.url }})"
           echo ::set-output name=ref::"$(echo $pr_info | jq -r '.head.ref')"
           echo ::set-output name=repo::"$(echo $pr_info | jq -r '.head.repo.full_name')"
+
       - name: Slash Command Dispatch
         id: scd
         uses: peter-evans/slash-command-dispatch@v2
         with:
-          token: ${{ secrets.SUPERTOPHER_PAT }}
+          token: ${{ env.PAT }}
```
Comment (supertopher) [inline .github/workflows/slash-commands.yml:33]: this is a real solid move @zzztimbo 

are all `env` vars hidden by default?

**Review by supertopher — APPROVED** [review]
This is a great fix Tim.  Appreciate the test runs in the PR as well 

ship it!

**Review by c-p-b — APPROVED** [review]
LGTM, but with the concern raised of token reusage tracked in https://github.com/airbytehq/airbyte-cloud/issues/2641

**Review by git-phu — APPROVED** [review]
nice!

also appreciate the links to the test runs!

File: .github/workflows/slash-commands.yml (hunk 1 above)
Comment (zzztimbo) [reply .github/workflows/slash-commands.yml:33]: Nope. Github recognizes the PAT as a secret, but probably not other strings?
image

**Pull request merged**